src/components/SwapWidget/SwapWidget.tsx

```diff
@@ -332,7 +332,11 @@ const SwapWidget: FC<SwapWidgetPropsType> = ({
   };
 
   const handleSetToken = (type: TokenSelectModalTypes, value: string) => {
-    const baseRoute = `${appRouteParams.justifiedBaseUrl}/${AppRoutes.swap}`;
+    const baseUrl =
+      appRouteParams.justifiedBaseUrl !== "/"
+        ? appRouteParams.justifiedBaseUrl
+        : "";
+    const baseRoute = `${baseUrl}/${AppRoutes.swap}`;
```

Comment:
Something wrong with appRouteParams?

Reply:
yes, you get // and an error

Reply:
Ah I see, it's broken.

Reply:
I fixed appRouteParams. Pushed as commit.

Reply:
great, I removed this piece of code now